core/agent.go

```diff
@@ -38,20 +38,31 @@ const (
 	activationCodeFlag      = "code"
 	activationIDFlag        = "id"
 	regionFlag              = "region"
-	orgIDFlag               = "org"
-	orgProviderFlag         = "orgProvider"
 	registerFlag            = "register"
 	versionFlag             = "version"
 	fingerprintFlag         = "fingerprint"
 	similarityThresholdFlag = "similarityThreshold"
-	bzeroInfoFlag           = "bzeroInfo"
+
+	// BZero const
+	bzeroInfoFlag       = "bzeroInfo"
+	bzeroAPIKeyFlag     = "apiKey"
```

Comment:
Id maybe call this `registrationKey` instead of apiKey

Reply:
0fcc5b4d